pages/people.md

```diff
@@ -30,6 +30,7 @@ based on commits.
 | [Alan Camillo](https://github.com/alanblueshift)         |                                     |            |
 | [Amal Krishnan](https://github.com/krishamal)            |                                     | [IST][ist] |
 | [Eric Newton](https://github.com/ericnewton)             |                                     | [ET][et]   |
+| [Furkan KAMACI](https://github.com/kamaci)               | [LAGOM] (https://www.lagom.ai)      | [TRT][trt] |
```

Comment:
Do not want a space after `[LAGOM]`.  Also need to add link for `[trt]` at end of page.